README.md

```diff
@@ -2,55 +2,256 @@
+
+The list of these tunnel-specific options is outlined in the table below.
 
 | Parameter            | Description                                                                                       | Type        | Required | Default                 |
 |----------------------|---------------------------------------------------------------------------------------------------|:-----------:|:--------:|-------------------------|
-| name                 | An unique, arbitrary name used to identify the connection.                                        | str         | yes      | -                       |
-| local                | Defines information specific to the local participant&#39;s public network interface. Only defined when the use case requires a single-host setup. | dict | no | - |
-| local/ip             | A key to the local dictionary defining the IP address of the local participant&#39;s public network interface. | str | no | - |
-| local/hostname       | A key to the local dictionary defining the DNS hostname of the local participant&#39;s public network interface | str | no | - |
-| local/subnet         | A key to the local dictionary defining the subnet of the local participant&#39;s public network interface. | str | no | - |
-| local/rsasigkey      | A key to the local dictionary defining the public key for RSA signature authentication.           | str         | no       | -                       |
-| local/cert           | A key to the local dictionary defining the certificate nickname of your certificate in the NSS database | str | no | - |
-| remote               | Defines information specific to the remote participant&#39;s public network interface. Only defined when the use case requires a single-host setup. | dict | no | - |
-| remote/ip            | A key to the remote dictionary defining the IP address of the remote participant&#39;s public network interface. | str | no | - |
-| remote/hostname      | A key to the remote dictionary defining the DNS hostname of the remote participant&#39;s public network interface | str | no | - |
-| remote/subnet        | A key to the remote dictionary defining the subnet of the remote participant&#39;s public network interface. | str | no | - |
-| remote/rsasigkey     | A key to the remote dictionary defining the public key for RSA signature authentication           | str         | no       | -                       |
-| remote/cert          | A key to the remote dictionary defining the certificate nickname of your certificate in the NSS database | str | no | - |
+| name                 | An unique, arbitrary name used to identify the connection.                                        | str         | no       | See [name](#name)       |
+| hosts                | A list of peers. A vpn tunnel will be constructed between each pair of hosts in this list. <br /> See [hosts](#hosts). | list | no | -               |
```

Comment:
It is a "list" but it is implemented as a `dict` where the keys are the names of the hosts from the Ansible inventory (this also helps guarantee that all of the hosts in a tunnel are unique), or otherwise name a host (for the case where a host in a tunnel is not in the Ansible inventory e.g. in a remote data center).  The values of the `dict` are the host specific parameters for that host.